tests/telemetry/test_payloads.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+
+from ddtrace.internal.telemetry.data.application import get_version
+from ddtrace.internal.telemetry.data.integration import create_integration
+from ddtrace.internal.telemetry.data.metrics import Series
+from ddtrace.internal.telemetry.data.payload import AppClosedPayload
+from ddtrace.internal.telemetry.data.payload import AppGenerateMetricsPayload
+from ddtrace.internal.telemetry.data.payload import AppIntegrationsChangedPayload
+from ddtrace.internal.telemetry.data.payload import AppStartedPayload
+from ddtrace.internal.telemetry.data.payload import Payload
+
+
+def test_app_started_payload():
+    payload = AppStartedPayload()
+
+    assert payload.request_type() == "app-started"
+    assert len(payload.dependencies) > 0
+    assert payload.to_dict() == {"dependencies": payload.dependencies}
+
+
+def test_app_closed():
+    payload = AppClosedPayload()
+
+    assert payload.request_type() == "app-closed"
+    assert payload.to_dict() == {}
+
+
+def test_app_integrations_changed():
+    integrations = [
+        create_integration("integration-1"),
+        create_integration("integration-2", enabled=False),
+        create_integration("integration-3", error="error"),
+    ]
+
+    payload = AppIntegrationsChangedPayload(integrations)
+
+    assert payload.request_type() == "app-integrations-changed"
+    assert len(payload.integrations) == 3
+    assert payload.to_dict() == {"integrations": integrations}
```

Comment:
you could assert on a static value for the integrations since they should be consistent based on `create_integration`.

the reason for this is if this payload expects a certain format for the integrations, but then we change up `create_integration`, we might break this payload.